Lib/glyphsLib/classes.py

```diff
@@ -1157,8 +1157,10 @@ class GSFontMaster(GSBase):
         "descender": float,
         "guideLines": GSGuideLine,
         "horizontalStems": int,
+        "iconName": str,
         "id": str,
         "italicAngle": float,
+        "name": str,
```

Comment:
Both master.iconName and master.name should be unicode, no?

Reply:
The `name` might be better of being unicode, but the iconName is not user accessible and only contains ascii.